_layouts/post-list.html

```diff
@@ -6,7 +6,7 @@
 {% for post in site.posts %}
   {% unless post.hidden or post.category == 'talk' or post.category == 'tutorial' %}
   <article class="post-item">
-    <time class="post-meta">{{ post.date | date: "%b %-d, %Y" }}</time>
+    <time class="post-meta">{{ post.date | date: "%d %b %Y" }}</time>
```

Comment:
It looks like the post list on the homepage needs updating as well (and fixing).